evap/static/ts/src/contact_modal.ts

```diff
@@ -0,0 +1,61 @@
+declare const bootstrap: typeof import("bootstrap");
+
+import { selectOrError, sleep, assert } from "./utils.js";
+import { csrfHeader } from "./csrf-utils.js";
+
+const success_message_timeout = 3000;
```

Comment:
Usually, global constants are written as `SCREAMING_SNAKE_CASE`:
```suggestion
const SUCCESS_MESSAGE_TIMEOUT = 3000;
```